docs/setup/docker/getting-started-docker.md

```diff
@@ -31,50 +37,42 @@ docker run \
   quay.io/nuclio/dashboard:stable-amd64
 ```
 
-> NOTE: _stable_ tag refers to the latest version released by nuclio from `master` branch (unlike versioned branches, e.g.: 1.3.x)
-
-Browse to `http://localhost:8070` to explore Nuclio
-
+<a id="deploy-a-function-with-the-nuclio-dashboard"></a>
 ## Deploy a function with the Nuclio dashboard
 
-Browse to `http://localhost:8070/projects/default/create-function`.
-You should see the [Nuclio dashboard](/README.md#dashboard) UI.
-Choose one of the built-in examples and click **Deploy**. 
-The first build will populate the local Docker cache with base images and other files, so it might take a while, depending on your network.
-When the function deployment is completed, you can click **Invoke** to invoke the function with a body.
+Browse to `http://localhost:8070` to see the [Nuclio dashboard](/README.md#dashboard).
+Select the "default" project and then select **New Function** from the action toolbar to display the **Create function** page (http://localhost:8070/projects/default/create-function).
+Choose one of the predefined template functions, and select **Deploy**. 
+The first build populates the local Docker cache with base images and other files, so it might take a while to complete, depending on your network.
+When the function deployment completes, you can select **Invoke** to invoke the function with a body.
```

Comment:
~invoke~ > _test_

Reply:
I've now merged the merged Minikube doc-update PR #1872 into the current PR branch, including replacing "Invoke" with "Test" — see commit 712a239.